source/_integrations/surepetcare.markdown

```diff
@@ -55,6 +55,21 @@ surepetcare:
     type: time
 {% endconfiguration %}
 
+## set_lock_state Service
+
+This service let you change the locking state of a flap
```

Comment:
```suggestion
This service let you change the locking state of a flap.
```

Reply:
Incorporated suggestion, and fixed 'This service let you'